### PR TITLE
maint-needed: epatch in EAPI6 cleanup

### PR DESCRIPTION
**Pull request by hanetzer**
Most are pretty simple. media-gfx/nvidia-texture-tools will require a new patch tarball hosted
somewhere, whether its a devspace or a distfile mirror, as the current one has several levels
of -p (-p0, -p1, and -p2 even).

### CODE REVIEW

File: media-gfx/nvidia-texture-tools/nvidia-texture-tools-2.0.8-r4.ebuild [hunk 1]
```diff
@@ -44,7 +45,7 @@ S="${WORKDIR}/${PN}"
 
 src_prepare() {
 	edos2unix cmake/*
-	EPATCH_SUFFIX=patch epatch "${WORKDIR}/patches"
+	EPATCH_SUFFIX=patch eapply "${WORKDIR}/patches"
```
Comment (SoapGentoo) [inline media-gfx/nvidia-texture-tools/nvidia-texture-tools-2.0.8-r4.ebuild]: tried adding `"${WORKDIR}/patches"` to `PATCHES` as last line?

File: x11-terms/tilda/tilda-0.9.6-r1.ebuild [hunk 2]
```diff
@@ -21,11 +21,8 @@ RDEPEND="x11-libs/vte:0
 DEPEND="${RDEPEND}
 	virtual/pkgconfig"
 
-src_prepare() {
-	epatch "${FILESDIR}"/${PN}-0.9.6-gdk_resources.patch
-	epatch "${FILESDIR}"/${PN}-0.9.6-glib-single-include.patch
-	epatch "${FILESDIR}"/${PN}-0.9.6-makefile.patch
-	eapply_user
-
-	eautoreconf
```
Comment (SoapGentoo) [inline x11-terms/tilda/tilda-0.9.6-r1.ebuild]: no more autotools?

File: media-gfx/nvidia-texture-tools/nvidia-texture-tools-2.0.8-r4.ebuild (hunk 1 above)
Comment (hanetzer) [reply media-gfx/nvidia-texture-tools/nvidia-texture-tools-2.0.8-r4.ebuild]: that may work, actually...

File: x11-terms/tilda/tilda-0.9.6-r1.ebuild (hunk 2 above)
Comment (hanetzer) [reply x11-terms/tilda/tilda-0.9.6-r1.ebuild]: Ack. Missed that, sorry. Fixed.

File: media-gfx/nvidia-texture-tools/nvidia-texture-tools-2.0.8-r4.ebuild (hunk 1 above)
Comment (hanetzer) [reply media-gfx/nvidia-texture-tools/nvidia-texture-tools-2.0.8-r4.ebuild]: Does work, nice. Will have to remember this in the future.

**Pull request closed without merging**